de.metas.acct.base/src/main/sql/postgresql/ddl/views/RV_Fact_Acct_Reposted_Documents.sql

```diff
@@ -0,0 +1,26 @@
+DROP VIEW IF EXISTS de_metas_acct.RV_Reposted_Documents ;
+
+
+CREATE OR REPLACE VIEW de_metas_acct.RV_Reposted_Documents 
+AS 
+SELECT 	
+
+pinst.AD_Client_ID,
+pinst.AD_Org_ID,
+pinst.Created,
+pinst.CreatedBy,
+pinst.Updated,
+pinst.UpdatedBy,
+pinst.IsActive,
+
+pinst.AD_PInstance_ID,
+to_number (regexp_matches(p_msg, '\d+')::text, '99999999999999999') AS AD_Table_ID ,
```

Comment:
Done.
Also added the where clause 
_AND substring(p_msg FROM 'AD_Table_ID[ =]*([0-9]*)')!='' AND substring(p_msg FROM 'Record_ID[ =]*([0-9]*)')!='' to the view